tools/code_format/check_format.py

```diff
@@ -118,10 +118,8 @@
     "./source/common/http/http2/codec_impl.h", "./source/common/http/http2/codec_impl.cc")
 
 RAW_TRY_ALLOWLIST = (
-    "./source/common/common/regex.cc",
-    "./source/common/common/thread.h",
-    "./source/common/network/utility.cc",
-)
+    "./source/common/common/regex.cc", "./source/common/common/thread.h",
+    "./source/common/network/utility.cc", "./source/common/network/dns_impl.cc")
```

Comment:
I think I have a slight preference for leaving dns_impl.cc out of this try-allowlist and the TRY_NEEDS_AUDIT there as I think there may be a path forward to fix the issue with the fuzz test.

In any case there is a TODO there.

Reply:
We already left comment and TODO in dns_impl.cc, explaining why we can't add main thread assertion to the try  block there.(Because a filter reuse that code in worker thread.) As long as we don't care about that filter, there is no security issue to fix.

Reply:
Yes I know it works, what you have. But what I makes me uncomfortable is that you are allow-listing a large file. 2 reasons not to love that:
 * other '`try` could be added to this file without triggering lint now
 * having this long list of allow-listed files makes it seem like it's OK to just grow that list.

So I think TRY_NEEDS_AUDIT in this one case is a better state to leave the system in. And hopefully we can address this by cleaning up the fuzz test.

Reply:
makes sense.